addons/account/tests/test_account_move_reconcile.py

```diff
@@ -2357,6 +2567,74 @@ def test_reconcile_rounding_issue(self):
 
         self.assertTrue(invoice.payment_state in ('in_payment', 'paid'))
 
+    def test_reconcile_plan(self):
+        @contextmanager
+        def rollback():
+            savepoint = self.cr.savepoint()
+            yield
+            savepoint.rollback()
+
+        comp_curr = self.company_data['currency']
+
+        line_1 = self._create_line_for_reconciliation(600.0, 600.0, comp_curr, '2017-01-01')
+        line_2 = self._create_line_for_reconciliation(-100.0, -100.0, comp_curr, '2017-01-02')
+        line_3 = self._create_line_for_reconciliation(700.0, 700.0, comp_curr, '2017-01-03')
+        line_5 = self._create_line_for_reconciliation(-700.0, -700.0, comp_curr, '2017-01-04')
+        line_4 = self._create_line_for_reconciliation(-500.0, -500.0, comp_curr, '2017-01-05')
```

Comment:
any reason to use different dates?

Reply:
just to prevent the sorted inside the reconciliation to change the order of lines. I want to keep the test predictive